code/player/TTTPlayer.Commands.cs

```diff
@@ -37,6 +41,55 @@ public static void RespawnPlayer(int id)
             }
         }
 
+        [ServerCmd(Name = "requestitem")]
```

Comment:
`spawnItem` maybe? or did old `GMOD`/`TTT2` use `requestitem`

Reply:
all commands are lowercase, that's why. But we could talk about `request_item` or `ttt_request_item`